pkg/model/project.proto

```diff
@@ -99,3 +105,49 @@ message ProjectRBACConfig {
     string editor = 2;
     string viewer = 3;
 }
+
+// ProjectTeam represents a team.
+message ProjectTeam {
+    // The name of team.
+    string name = 1 [(validate.rules).string.min_len = 1];
+    // The name of rbac role.
+    string role_name = 2 [(validate.rules).string.min_len = 1];
+}
+
+// ProjectRBACRole represents a RBAC role.
+message ProjectRBACRole {
+    // The name of role.
+    string name = 1 [(validate.rules).string.min_len = 1];
+    // The policy of role.
+    repeated ProjectRBACPolicy policy = 2;
+}
+
+message ProjectRBACPolicy {
+    enum ResourceType {
+      APPLICATION = 0; 
+      DEPLOYMENT = 1;
+      EVENT = 2;
+      PIPED = 3;
+      DEPLOYMENTCHAIN = 4;
+      PROJECT = 5;
+      APIKEY = 6;
+      INSIGHT = 7;
+    }
+
+    enum ActionType {
+      GET = 0;
+      LIST = 1;
+      CREATE = 2;
+      UPDATE = 3;
+      DELETE = 4;
+    }
+
+    // The type of resource.
+    ResourceType resource_type = 1 [(validate.rules).enum.defined_only = true];
+    // The type of action.
+    ActionType action_type = 2 [(validate.rules).enum.defined_only = true];
```

Comment:
And this should be a list of actions. 🤔

Reply:
Here, `action_type` and `label` are defined to be 1:1.
This is so that different action_type permissions can be given to each label resource.

For instance, There is a case which somebody wanna give developer all action permission to control some applications in the dev env but don't wanna give that in the prod env.

But it's complicated a little bit, that's why,  I think it also good that we don't support this one.

Reply:
🤔 
With this data format, I wonder how can we re-generate the original text of the policy to show on the web?

Reply:
I assumed we'll compares whether the actions associated with the label match for the same resource type and decide the format. If they match, it's displayed as shown ①. Otherwise, it's displayed as shown ②.
`
① 
application{env: dev, app: v1}, get/list/create

② 
application{env: dev}, get/list/create
application{app: v1}, get/list/create/update/delete
`

Reply:
@nghialv 
Thank you for your reviews.
I fixed some points. so PTAL when you have enough time.